package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "start": "node .",
     "pretest": "jshint .",
-    "posttest": "nsp check"
+    "posttest": "nsp check",
+    "api": "node .",
+    "client": "./node_modules/.bin/webpack-dev-server --config server/webpack.dev.config.js"
```

Comment:
Thanks for the heads up on the webpack-dev-server command.

The added config in server/webpack.dev.config.js appears to be necessary. I can't see the local server without it.